### PR TITLE
Adds N/S hemisphere flag for northing values to UTM coordinates

### PR DESCRIPTION
**Pull request by coyotesqrl**
Also simplifies getUtmLetterDesignator method.

These changes are analogues to these changes made to [usng.js](https://github.com/codice/usng.js)

* https://github.com/codice/usng.js/pull/11
* https://github.com/codice/usng.js/pull/15

### CODE REVIEW

File: usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java [hunk 1]
```diff
@@ -63,6 +63,12 @@
      */
     Character getLattitudeBand();
 
+    /**
+     *
+     * @return the N/S indicator if using northing values with hemisphere indicator
+     */
+    Character getNSIndicator();
```
Comment (stustison) [inline usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java]: ✏️ We should probably have constants somewhere to define what values this character can be.

**Review by stustison — APPROVED** [review]

File: usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java (hunk 1 above)
Comment (stustison) [reply usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java]: Or use an enum...

Comment (coyotesqrl) [reply usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java]: I sort of want to get rid of the public constructors and move to factory methods with an enum, but that felt a bit much for this PR.

Comment (coyotesqrl) [reply usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java]: But maybe I'll just tackle that now.

File: usng4j-api/src/main/java/org/codice/usng4j/NSIndicator.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ **/
+
+package org.codice.usng4j;
+
+/**
+ * Indicator for North/South for UTM coordinates
+ */
+public enum NSIndicator {
```
Comment (coyotesqrl) [inline usng4j-api/src/main/java/org/codice/usng4j/NSIndicator.java:27]: Added an enum; didn't swap out ctors.

**Review by dcruver — APPROVED** [review]

**Pull request merged**